content/docs/getting_started/report.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Report"
+description = "Open mSupply Reports."
+date = 2022-03-19T18:20:00+00:00
+updated = 2022-03-19T18:20:00+00:00
+draft = false
+weight = 31
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
++++
+
+## Reports
+
+The reports page gives you an overview of the reports available in Open mSupply.
+You can access the reports page by clicking on the `Reports` menu item.
+![goto_report](/docs/getting_started/images/goto_report.png)
+
+You will be presented with a list of reports that you can run. The reports
```

Comment:
wondering about this - the verb 'run' here sounded technical, and we use 'generate' later on 🤔 
```suggestion
You will be presented with a list of reports that you can generated. The reports
```